shell/testing/support.h

```diff
@@ -0,0 +1,5 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#define SUPPORT_FRACTIONAL_TRANSLATION 1;
```

Comment:
I think this is how C++ works 😄

Reply:
maybe I actually need to include it in the other main. Then ... how do I do that conditionally via GN. I need to learn gn